src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -19,7 +19,7 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
 
     lazy val err1 = ClusterError("some failure", 10, Instant.now().truncatedTo(ChronoUnit.SECONDS))
     lazy val c1UUID = UUID.randomUUID()
-    lazy val createdDate = Instant.now()
+    lazy val createdDate, dateAccessed = Instant.now()
```

Comment:
FYI: It doesn't seem like the `lazy`ness of the `val`s above buys us anything since the code where they're used (e.g. `c1`) is strictly evaluated.